src/main/java/core/selenium/helpers/Helper.java

```diff
@@ -0,0 +1,23 @@
+package core.selenium.helpers;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Helper class.
+ */
+public final class Helper {
+
+    private Helper() {
+
+    }
+
+    /**
+     * This method return a date using the format yyyy-MM-dd HH:mm:ss.
+     * @return simpleDateFormat
+     */
+    public static String getTime() {
```

Comment:
if you are using this time for UniqueId maybe you should change the name of the method, with that name I expect it returns current time